civicaseextras.civix.php

```diff
@@ -495,3 +508,62 @@ function (phpQueryObject $doc) {
       });
   $angular->add($changeSet);
 }
+
+/**
+ * Adds the case duration column and value to the case list table template of civicase.
+ *
+ * @param Manager $angular as provided by the alter angular hook.
+ */
+function _civicaseextras_civicrm_alterAngular__addCaseDurationToCaseList (\Civi\Angular\Manager &$angular) {
```

Comment:
All these functions make the file a lot longer, and the method names are very long. Can you put this logic in a class instead, maybe something like `CaseDurationAngularModifier`?